README.md

```diff
@@ -51,7 +50,13 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza {
+  temperature: 70,
+  bake: function (temperature) {
+    temperature + 230;
+    return temperature;
+  }
+}
```

Comment:
this works! but you should rely on `pizza.temperature` instead of a function argument here. Consider the following code:

```js
console.log( pizza.temperature )
pizza.bake(350)
console.log( pizza.temperature )
```